tests/unix_listener.rs

```diff
@@ -89,6 +89,7 @@ fn unix_listener_register() {
 }
 
 #[test]
+#[cfg_attr(target_os = "cygwin", ignore)]
```

Comment:
Why are these ignored?

Reply:
The `connect` method always blocks because there's no `accept`. Thus, the spawned thread never waits for the barrier, making the testing thread block.